### DISCUSSION TITLE
How to deploy to gitpod

### DISCUSSION THREAD

**Discussion by RebbePod** [aahnik/tgcf#334] [Q&A]
I looking for the best free option for live mode, as heroku stopped working for me.

Getting stuck on instruction #4 
> Create two new files .env and tgcf.config.yml and fill them up.
Where do I create those files and how do I set them up?

**Comment by simonlkch** [comment]
you need 24x7?
gitpod has 50 hrs/month free only

[https://www.gitpod.io/pricing](https://www.gitpod.io/pricing)

**Reply by RebbePod** [reply]
Your right, I guess I'll have to use heroku, I think the reason for it not working may be because I'm trying to use a whitelist with an explanation mark
